docs/content/3.components/table.md

```diff
@@ -213,6 +213,9 @@ props:
   class: 'flex-1 max-h-[312px]'
 ---
 ::
+### WatchDeep
```

Comment:
I don't think it's necessary to add this section, the prop will be documentation in the `### Props` anyway.